xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/hierarchy_macros.vm

```diff
@@ -163,7 +163,10 @@
         #set ($closingTag = '')
       #end
       class="$stringtool.join($classNames, ' ')">#if ($item.url)<a href="$item.url"
-        >#end$item.label#if ($item.url)</a>#end$treeNavigation$closingTag##
+      #if ($foreach.index == 0 && !$options.plain)
+        title="$escapetool.xml($services.localization.render('core.menu.type.home'))"
```

Comment:
I noticed that you have reused the `core.menu.type.home` translation key. Are you sure it has the same meaning as when it was first introduced?